cfme/tests/cloud_infra_common/test_vm_ownership.py

```diff
@@ -126,15 +128,23 @@ def test_user_ownership_crud(request, user1, setup_provider, provider):
         assert user_ownership_vm.exists, "vm not found"
     user_ownership_vm.unset_ownership()
     with user1:
-        assert not user_ownership_vm.exists, "vm exists"
+        try:
```

Comment:
**Required**: We have a code duplication here. Better to move it to function with return value True/False. And then assert the result of this function in the test.